scripts/make.go

```diff
@@ -236,11 +236,17 @@ func buildFlags() []string {
 	if err != nil {
 		log.Fatal(err)
 	}
+	gctrimpath := "-gcflags=all=-trimpath=${PWD}"
```

Comment:
I don't want this always on (or even by default).

Reply:
How do you want this implemented in that case? There is no good reason to keep compilation paths in the binary.

https://reproducible-builds.org/docs/build-path/

Reply:
I'm ok with having a `-trimpath` flag.